tests/security_features_test.go

```diff
@@ -310,6 +310,46 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 			}, 30*time.Second, 10*time.Second).Should(BeNil())
 		})
 	})
+	Context("The VMI SELinux context status", func() {
+		It("Should get set and stay the the same after a migration", func() {
+			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+
+			By("Starting a New VMI")
+			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
+			Expect(err).NotTo(HaveOccurred())
+			libwait.WaitForSuccessfulVMIStart(vmi)
+
+			By("Ensuring VMI is running by logging in")
+			libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
+
+			By("Ensuring the VMI SELinux context status gets set")
+			seContext := ""
+			Eventually(func() string {
+				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Get(context.Background(), vmi.Name, &metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				seContext = vmi.Status.SelinuxContext
+				return seContext
+			}, 30*time.Second, 10*time.Second).ShouldNot(BeEmpty(), "VMI SELinux context status never got set")
+
+			By("Ensuring the VMI SELinux context status matches the virt-launcher pod files")
+			stdout := tests.RunCommandOnVmiPod(vmi, []string{"ls", "-lZd", "/"})
+			Expect(stdout).To(ContainSubstring(seContext))
+
+			By("Migrating the VMI")
+			checks.SkipIfMigrationIsNotPossible()
```

Comment:
I see multiple 
`SigComputeMigrations = []interface{}{Label("sig-compute-migrations")}`
or 
`RequiresTwoSchedulableNodes = []interface{}{Label("requires-two-schedulable-nodes")}`

Reply:
`SigComputeMigrations` is the for the horribly named lane that runs 3 nodes, so that there will be 2 with CPUManager, which is required only for migrating VMIs with dedicated CPUs, which doesn't apply here.
I did miss the recently added `RequiresTwoSchedulableNodes` which should indeed do the trick, will add, thanks!

Reply:
Thank you! I really appreciate it as we need to clean this up! I was calling for more sophisticated labels and wait a bit before we start introducing them but well we did not